src/server/chromium/chromium.ts

```diff
@@ -173,4 +173,5 @@ const DEFAULT_ARGS = [
   '--enable-automation',
   '--password-store=basic',
   '--use-mock-keychain',
+  '--blink-settings=primaryHoverType=2,availableHoverTypes=2,primaryPointerType=4,availablePointerTypes=4'
```

Comment:
Perhaps, only for headless? Won't this break my touchpad on mac headful?

Reply:
Sure. A touchpad on a mac doesn't count as a course pointer, but I believe a touchscreen on a windows laptop does. I made it headless only. This will make the test fail in headful mode on a windows laptop with a touchscreen, but I don't think that's a big deal.